pages/api/transfer/events.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { google } from "googleapis";
+
+import { IEventDTO } from "../../../dtos";
+import { getEvents } from "../../../utils";
+
+const API = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
This const should also have a `try` and `catch` to handle potential errors that might occur during the interaction with the Google Sheets API